File: prototypes/Raptor/DNAustria.Backend/src/DNAustria.Backend/Models/Enums.cs
namespace DNAustria.Backend.Models;

public enum Classification
{
    Scheduled,
    OnDemand
}

public enum EventStatus
{
    Draft,
    Approved,
    Transferred
}

public enum TargetAudience
{
    Vorschulkinder = 10,
    Schulkinder = 20,
    Jugendliche_SekI = 30,
    Jugendliche_Berufsschulen = 40,
    Jugendliche_SekII = 50,
    Erwachsene = 60,
    Familien = 70,
    Nur_Maedchen_Frauen = 80
}

public enum EventTopic
{
    Digitalisierung_IT = 100,
    Kunst_Kultur = 200,
    Sprachen_Literatur = 300,
    Medizin_Gesundheit = 400,
    Geschichte_Demokratie_Gesellschaft = 500,
    Wirtschaft_Recht = 600,
    Naturwissenschaft_Klima_Umwelt = 700,
    Mathematik_Zahlen_Daten = 800
}

File: prototypes/Gemini/DNAustria/frontend/src/app/models/enums.ts
export enum TargetAudience {
    VorschulkinderElementarstufe = 10,
    SchulkinderPrimarstufe = 20,
    JugendlicheSekundarstufeI = 30,
    JugendlicheBerufsschulenPTS = 40,
    JugendlicheSekundarstufeII = 50,
    Erwachsene = 60,
    Familien = 70,
    NurMaedchenFrauen = 80
}

export enum EventTopic {
    DigitalisierungKuenstlicheIntelligenzITTechnik = 100,
    KunstKultur = 200,
    SprachenLiteratur = 300,
    MedizinGesundheit = 400,
    GeschichteDemokratieGesellschaft = 500,
    WirtschaftRecht = 600,
    NaturwissenschaftKlimaUmwelt = 700,
    MathematikZahlenDaten = 800
}

export enum EventStatus {
    Draft = 0,
    Approved = 1,
    Transferred = 2
}

export enum EventClassification {
    Scheduled = 0,
    OnDemand = 1
}
